lib/create-plugin.js

```diff
@@ -356,16 +370,24 @@ function getDeepestUniqueSelection(definition) {
       return IGNORED_PATH_FIELDS.indexOf(currentSelection.name.value) < 0
     })
 
-    if (filtered.length > 1 || filtered.length === 0) {
+    if (filtered.length === 0 || filtered.length > 1) {
       // selections not unique OR
       // only one IGNORED_PATH_FIELDS item in selections
       break
     }
 
     selection = filtered[0]
 
-    // Inline fragments do not have a name.
-    selection.name && deepestPath.push(selection.name.value)
+    if (isNamedType(selection)) {
+      const lastItemIdx = deepestPath.length - 1
+      // add type to the last item in deepestPath array
+      // (i.e - `_entities<Human>`)
+      deepestPath[lastItemIdx] =
+        `${deepestPath[lastItemIdx]}<${selection.typeCondition.name.value}>`
+    } else {
+      // Inline fragments do not have a name.
```

Comment:
this comment is prob no longer applicable, thoughts?

Reply:
Yeah, I think we can remove comment and leave defensive check, `selection.name &&`

Reply:
I may just remove the comment. i worry there are other types that lack name